share/picongpu/examples/LaserWakefield/etc/picongpu/8.cfg

```diff
@@ -31,29 +30,48 @@
 ## Section: Required Variables ##
 #################################
 
-TBG_wallTime="2:00:00"
+TBG_wallTime="1:00:00"
 
 TBG_devices_x=2
-TBG_devices_y=4
-TBG_devices_z=1
+TBG_devices_y=2
+TBG_devices_z=2
 
-TBG_gridSize="192 1024 192"
-TBG_steps="4000"
-
-# leave TBG_movingWindow empty to disable moving window
-TBG_movingWindow="-m"
```

Comment:
could you please re-add the sliding window. Maybe you should add again 4 GPUs in Y direction to have a larger window.